src/browsers/tob/shared/components/ManhattanPlotNew.js

```diff
@@ -0,0 +1,362 @@
+                      {numberWithCommas(tick)}
+                    </text>
+                    <line y2={6} stroke="black" />
+                  </g>
+                ))}
+              </g>
+              <line y2={`${innerHeight}`} stroke="black" />
+              {yScaleLocal.ticks(4).map((tick) => (
+                <g key={tick} transform={`translate(0, ${yScaleLocal(tick)})`}>
+                  <text key={tick} style={{ textAnchor: 'end' }} x={-6} dy=".32em">
+                    {tick}
+                  </text>
+                  <line x2={-3} stroke="black" />
+                  <line
+                    x2={innerWidth}
+                    stroke={yScaleLocal(tick) === innerHeight ? 'none' : 'lightgrey'}
+                  />
+                </g>
+              ))}
+              <g ref={brushRef} />
+              <g clipPath="url(#clipManhattanPlot)">
+                {data.map((d) => (
+                  <React.Fragment key={`${accessors.x(d)},${accessors.y(d)},${accessors.color(d)}`}>
+                    {(accessors.isSelected(d) || accessors.isReference(d)) && (
+                      <line
+                        key={`${accessors.x(d)},${accessors.y(d)},${accessors.color(
+                          d
+                        )}SelectedLine`}
+                        x1={xScaleLocal(accessors.x(d))}
+                        x2={xScaleLocal(accessors.x(d))}
+                        y1={0}
+                        y2={innerHeight}
+                        strokeDasharray={12}
+                        stroke="black"
+                        opacity="0.5"
+                      />
+                    )}
+                    {accessors.isReference(d) ? (
+                      <g
+                        transform={`translate(${xScaleLocal(accessors.x(d))}, ${yScaleLocal(
+                          accessors.y(d)
+                        )}),rotate(45)`}
+                      >
+                        <rect
+                          key={`${accessors.x(d)},${accessors.y(d)},${accessors.color(d)}`}
+                          width={10}
+                          height={10}
+                          fill={defaultCellTypeColors()[accessors.color(d)]}
+                          onClick={onClick}
+                          // onMouseOver={() => onMouseOver()}
+                          // onFocus={() => onMouseOver()}
+                          // onMouseMove={(e) => onMouseMove(e, d)}
+                          // onMouseLeave={() => onMouseLeave()}
+                        />
+                      </g>
+                    ) : (
+                      <circle
+                        key={`${accessors.x(d)},${accessors.y(d)},${accessors.color(d)}`}
+                        cx={xScaleLocal(accessors.x(d))}
+                        cy={yScaleLocal(Math.min(4, -Math.log10(accessors.y(d))))}
+                        r={accessors.isSelected(d) ? 6 : 3}
+                        fill={defaultCellTypeColors()[accessors.color(d)]}
```

Comment:
Color accessor should be returning a hex string and not a cell type id because the plot should be generalisable to other types of data. Access to `defaultCellTypeColors` should be occurring in the parent component.